config/default.json

```diff
@@ -6,6 +6,10 @@
             "url": "mongodb://localhost/fizca"
         }
     },
+    "session": {
+        "max_age_millis": 86400000,
```

Comment:
Really wish json allowed commenting so you could state how long this is in human readable format...

Reply:
yeeeep... SO there is this format called json5 which allows comments and other goodies. 

the `config` package also allow for `.js` configuration files as well as YAML and some other formats: 

https://github.com/lorenwest/node-config/wiki/Configuration-Files

Historically i've always used `.json` because i feel configurations should be that, but I'm willing to change if you're willing to put the PR :D